.gitignore

```diff
@@ -34,8 +34,3 @@ build/
 ### Mac OS ###
 .DS_Store
 .idea/
-
-src/main/HTTP Checks/
-/dependency-reduced-pom.xml
```

Comment:
Warum hast du die entfernt?

Reply:
Weil es in der main nicht mehr vorhanden ist. Ich habe dort nichts verändert